### PR TITLE
Travis-Change

### PR DESCRIPTION
**Pull request by yonikashi**

### CODE REVIEW

**Comment by yonikashi** [comment]
This change in travis Yaml is a begining for our CI process.
This change run a test with our Blockchain-ops repo inside a docker.
Later will be able to push it to docker hub.

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,19 @@
-language: cpp
-sudo: false
+language: python
+python:
+- 3.7-dev
+os: linux
+dist: xenial
```
Comment (oryband) [inline .travis.yml]: change this to 18.04

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,19 @@
-language: cpp
-sudo: false
+language: python
+python:
+- 3.7-dev
```
Comment (oryband) [inline .travis.yml]: not critical but still:
- indent nested lists by 2 spaces. also apply for the rest of this file
- run http://www.yamllint.com/ on this file

File: .travis.yml [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+  - BUILD_TYPE="stellar-core"
 
-compiler:
-- clang
-- gcc
+#language: cpp
+#sudo: false
+#compiler:
+#- clang
+#- gcc
```
Comment (oryband) [inline .travis.yml]: remove this part, or explain why its commented out

File: .travis.yml [hunk 4]
```diff
@@ -1,9 +1,19 @@
...
 
 addons:
   postgresql: "9.3"
```
Comment (oryband) [inline .travis.yml]: should be 9.5

File: .travis.yml [hunk 5]
```diff
@@ -26,8 +36,18 @@ addons:
     - pkg-config
     - clang-format-5.0
```
Comment (oryband) [inline .travis.yml]: we don't really need this package. only what's in here https://github.com/kinecosystem/blockchain-ops/blob/40ccfc49a18693f09d054c3ece5c194a01f67063/images/dockerfiles/Dockerfile.stellar-core-build#L4

File: .travis.yml [hunk 6]
```diff
@@ -26,8 +36,18 @@ addons:
...
     - pandoc
+    - libffi-dev
 
-script: ./travis-build.sh
+before_script:
+- git describe --tags >> describe.txt
+- export MY_TAG="$(cat describe.txt)"
```
Comment (oryband) [inline .travis.yml]: - in stellar's repo this use `--always` as well: https://github.com/stellar/stellar-core/blob/0edfee2c6325165f3a9cbe2be7371e98f139b082/travis-build.sh#L16
- and when setting the version in the binary they also use `--dirty`: https://github.com/stellar/stellar-core/blob/43bd5ecbab85c397c5e9af18f2c87520cea33b7e/src/Makefile.am#L47

File: .travis.yml [hunk 7]
```diff
@@ -26,8 +36,18 @@ addons:
...
+
+script:
+- mkdir $COMMIT && cd $COMMIT
+- git clone https://github.com/kinecosystem/blockchain-ops.git && cd blockchain-ops
+- pip -V
+- pipenv sync
+- pipenv run invoke build-core --branch="master" --version=$MY_TAG
```
Comment (oryband) [inline .travis.yml]: the branch here should be the current branch this build is run on

File: .travis.yml (hunk 2 above)
Comment (yonikashi) [reply .travis.yml]: File is intact... Checked that

**Comment by yonikashi** [comment]
Hey Ori,
Please review again my PR.
Did the changes.

File: .travis.yml (hunk 1 above)
Comment (yonikashi) [reply .travis.yml]: Travis still does not support Latest ubuntu.
But this is only the platform for running the docker. Not the build itself.
The build platform version is defined in Blockchain-ops repo.

File: .travis.yml (hunk 6 above)
Comment (oryband) [reply .travis.yml]: @yonikashi what about this comment?

**Pull request merged**